src/server/server.ts

```diff
@@ -270,6 +404,10 @@ registerRoute('get', '/version', (req, res) => {
 function compileCommon(req: express.Request, res: express.Response, wrapWithRunnerChrome?: boolean) {
     const data: IRunnerState = JSON.parse(req.body.data);
     const { snippet, returnUrl } = data;
+    let isTrustedSnippet = req.body.isTrustedSnippet;
```

Comment:
Isn't this just the same as `req.body.isTrustedSnippet || false`, without the need for an "if" check below?